docker-compose.prod.yml

```
@@ -0,0 +1,75 @@
+      
+      # Email validation settings
+      - OAUTH2_PROXY_EMAIL_DOMAINS=speedandfunction.com
+      
+      # Required settings
+      - OAUTH2_PROXY_COOKIE_DOMAINS=.temporal.speedandfunction.com
+      - OAUTH2_PROXY_WHITELIST_DOMAINS=.temporal.speedandfunction.com
+      - OAUTH2_PROXY_HTTP_ADDRESS=0.0.0.0:80
+      - OAUTH2_PROXY_UPSTREAMS=http://temporal-ui:80
+      - OAUTH2_PROXY_REDIRECT_URL=${OAUTH2_PROXY_REDIRECT_URL:-http://temporal.temporal.speedandfunction.com/oauth2/callback}
+      - OAUTH2_PROXY_COOKIE_SECRET=${OAUTH2_PROXY_COOKIE_SECRET:-cookie-secret-1234567890}
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)